modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -309,6 +309,24 @@ OR
 |The SFTP server used by `must-gather-operator` to upload diagnostic logs to help troubleshoot issues with the cluster.
 |===
 
+. Allowlist the following URLs for optional third-party content:
++
+[cols="6,1,6",options="header"]
+|===
+|Domain | Port | Function
+|`registry.connect.redhat.com`
+| 443
+| Required for all third-party-images and certified operators.
+
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
+| 443
+| Provides access to container images hosted on `registry.connect.redhat.com`
+
+|`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
+| 443
+| Required for Sonatype Nexus, F5 Big IP operators.
```

Comment:
Not 100% sure on the naming of either Operator, suggestions based on [this page for Nexus](https://support.sonatype.com/hc/en-us/articles/4757606085523-How-to-install-Nexus-Pro-using-Nexus-Operator-in-OpenShift) and [this page for F5](https://community.f5.com/t5/technical-articles/using-f5-big-ip-controller-operator-for-openshift/ta-p/288220).

```suggestion
| Required for the Nexus Repository Operator and F5 BIG-IP Operator.
```